src/command.ts

```diff
@@ -136,14 +137,12 @@ function takeScreenshot(
   screenshotOptions?: ScreenshotOptions
 ): Cypress.Chainable<string> {
   const objToOperateOn = subject !== undefined ? cy.get(subject) : cy
-
   let screenshotPath: string
   return (
     objToOperateOn
       .screenshot(name, {
         ...screenshotOptions,
-        // eslint-disable-next-line
-        onAfterScreenshot(_el: JQuery, props: any) {
+        onAfterScreenshot(_el, props) {
```

Comment:
We don't need to define the type of the parameters (_el & props) because they are already typed